src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapeGame.java

```diff
@@ -762,6 +785,30 @@ public void policeInspectedPrisioner(String policeName, String prisionerName) {
         }
     }
 
+    /**
+     *
+     * @param player
+     * @param itemHeld
+     * @param door
```

Comment:
Do not add @param if you are not going to give them a description. I am not saying to give them a description. I prefer that you don't mention the parameters.